molecule/commands.py

```diff
@@ -159,6 +159,8 @@ def converge(self, idempotent=False, create_instances=True, create_inventory=Tru
             kwargs.pop('_err', None)
             kwargs['_env']['ANSIBLE_NOCOLOR'] = 'true'
             kwargs['_env']['ANSIBLE_FORCE_COLOR'] = 'false'
+            kwargs['_env']['ANSIBLE_CALLBACK_PLUGINS'] = os.path.join(
+                sys.prefix, 'share/molecule/ansible/plugins/callback/idempotence')
```

Comment:
@abrown-sg The updated idempotence test, already falls back to the default behaviour. It does not display a warning though. This is a good idea. 

Again, I think the user should not have the control of the ansible output for the tests.